DA-216: Add go-to-last-edited-document-button in projects explorer

JeannedArk opened the pull request:
The button is only for annotators visible and opens the last edited
document depending on the last edit properties.

JeannedArk commented:
@judembo @stgrue Updated

judembo commented:
looks good and works fine, but would it be possible to have the "hand" symbol on mouseover as for other links?

JeannedArk committed:
DA-216: last-edited-document column in projects explorer

The column displays the last edited document depending on the users state
and last edited value and links to the last edited document. The column
is only for annotators visible.
